File: LV1/1.5/tri/tri.py
op = ['+', '-', '*', '/']


def find_sign(a, b, c):
    for p in op:
        if eval(f"{a}{p}{b}") == c:
            return (p, '=')
        if eval(f"{b}{p}{c}") == a:
            return ('=', p)


a, b, c = [int(i) for i in input().split(' ')]
sign = find_sign(a, b, c)
print(f"{a}{sign[0]}{b}{sign[1]}{c}")

File: LV1/1.5/tri/tri.rb
def find_sign(a, b, c)
  op = ['+', '-', '*', '/']

  op.each do |p|
    return [p, '='] if eval("#{a}#{p}#{b}") == c
    return ['=', p] if eval("#{b}#{p}#{c}") == a
  end
end

a, b, c = gets.split.map(&:to_i)
sign = find_sign(a, b, c)
puts "#{a}#{sign[0]}#{b}#{sign[1]}#{c}"
